### PR TITLE
[Frontend] Prettier config to be consistent with existing code style

### PR DESCRIPTION
**Pull request by Bobgy**
Config is proposed here: https://github.com/kubeflow/pipelines/issues/2406#issuecomment-542640923

/area front-end

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2409)

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -53,3 +56,30 @@ this image, you'll need to port forward 3000, and pass the environment
...
 you can run using `npm run api` separately.
+
+## Code Style
+
+We use [prettier](https://prettier.io/) for code formatting, our prettier config
+is [here](https://github.com/kubeflow/pipelines/blob/master/frontend/.prettierrc.yaml).
+
+To understand more what prettier is: [What is Prettier](https://prettier.io/docs/en/index.html).
+
+### IDE Integration
```
Comment (rmgogogo) [inline frontend/README.md]: Not sure whether it has conflict with VSCode setting for Pantheon.

If yes, configure two IDEs may not be good. In such case, we may revisit this part later. Maybe just reuse Pantheon setting?

Comment (Bobgy) [reply frontend/README.md]: I think there are a few problems with pantheon setting (clang-format):
1. It is very different from existing code base
2. It has no support for tsx (react)

Comment (Bobgy) [reply frontend/README.md]: > If yes, configure two IDEs may not be good. In such case, we may revisit this part later. Maybe just reuse Pantheon setting?

We can co-maintain a vscode [workspace setting](https://code.visualstudio.com/docs/getstarted/settings) just for this repo.

**Pull request merged**